internal/policy/commit/commit.go

```diff
@@ -27,6 +27,10 @@ type Commit struct {
 	// Imperative enforces the use of imperative verbs as the first word of a
 	// commit message.
 	Imperative bool `mapstructure:"imperative"`
+	// AnyScope allows any scope and ignores Commit.Conventional.Scopes. This
+	// is useful if you have a very large repository with too many scopes than
+	// can be kept track of.
+	AnyScope bool `mapstructure:"anyScope"`
```

Comment:
I think this makes more sense to be in the `Conventional` struct since this options applies only to that.